edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
```

Comment:
Hm, not an easy task. Possibly a PR for the student experience that included the message for navigating away from the exam? There are actually a couple of PRs that merged still with open questions from me (and thus incomplete UI string review), but I figured there would be a chance to do one final pass closer to the end.